umh-core/pkg/service/redpanda/redpanda.go

```diff
@@ -956,3 +962,133 @@ func formatMemory(memory int) string {
 
 	return fmt.Sprintf("%d%s", memory, units[unitIndex])
 }
+
+// updateRedpandaClusterConfig updates Redpanda's cluster config via HTTP PUT request
```

Comment:
can you add into the docstring explaining why this works? we've put a lot of time and information in the research and tried different methods.

We should explain all of the methods (with restart), and as far as I remember this here is that we enforce the setting by reading it back immediately?

Reply:
other options were also:
1. do the restart by setting a EventDone
2. have this done via an S6 service with curl

but what doesn ow happen for things that DO require a restart according to redpanda? I think currently we can only change the ones without requiring a restart with the read back trick